Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * AST: Export NatSpec comments above each statement as their documentation.
  * Optimizer: Simple inlining when jumping to small blocks that jump again after a few side-effect free opcodes.
+ * Yul EVM Code Transform: Stack Optimization: Defer allocating stack slots for return variables until after the first expression statements and assignments that do not reference them.
```

Comment:
after the first that do not reference them? So it's always the second?

Reply:
Right, it should be ``after any``.

Reply:
Should I also squeeze in something about potentially popping arguments in there?